UNRELEASED.md

```diff
@@ -14,6 +14,9 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 ### Bug fixes
 
+- Fixed accessibility issue with `Autocomplete` where keyboard navigation of options was laggy and skipped options([#1346](https://github.com/Shopify/polaris-react/issues/1346))
```

Comment:
```suggestion
- Fixed accessibility issue with `Autocomplete` where keyboard navigation of options was laggy and skipped options([#1887](https://github.com/Shopify/polaris-react/pull/1887))
```

Reply:
We reference the pull request that made the changes 😄

Reply:
oops! haha! sorry! 🤦‍♀